test/integration/db_integration_test.go

```diff
@@ -0,0 +1,108 @@
+// Copyright 2020 Insolar Network Ltd.
+// All rights reserved.
+// This material is licensed under the Insolar License version 1.0,
+// available at https://github.com/insolar/block-explorer/blob/master/LICENSE.md.
+
+// +build heavy_mock_integration
+
+package integration
+
+import (
+	"context"
+	"io"
+	"testing"
+	"time"
+
+	"github.com/insolar/block-explorer/etl/extractor"
+	"github.com/insolar/block-explorer/etl/models"
+	"github.com/insolar/block-explorer/etl/processor"
+	"github.com/insolar/block-explorer/etl/storage"
+	"github.com/insolar/block-explorer/etl/transformer"
+	"github.com/insolar/block-explorer/etl/types"
+	"github.com/insolar/block-explorer/testutils"
+	"github.com/insolar/block-explorer/testutils/connection_manager"
+	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
+)
+
+type dbIntegrationSuite struct {
+	suite.Suite
+	c connection_manager.ConnectionManager
+}
+
+func (a *dbIntegrationSuite) SetupSuite() {
+	a.c.StartGrpc(a.T())
+	a.c.StartDB(a.T())
+}
+
+func (a *dbIntegrationSuite) TearDownSuite() {
+	a.c.Stop()
+}
+
+func (a *dbIntegrationSuite) TestGetRecordsFromDb() {
+	recordsCount := 10
+	expRecords := testutils.GenerateRecordsList(recordsCount)
+	stream, err := a.c.ImporterClient.Import(context.Background())
+	require.NoError(a.T(), err)
+	for _, record := range *expRecords {
+		if err := stream.Send(&record); err != nil {
+			if err == io.EOF {
+				break
+			}
+			a.T().Fatal("Error sending to stream", err)
+		}
+	}
+	reply, err := stream.CloseAndRecv()
+	require.NoError(a.T(), err)
+	require.True(a.T(), reply.Ok)
+
+	require.Len(a.T(), a.c.Importer.GetSavedRecords(), recordsCount)
+
+	ctx := context.Background()
+	extractorMn := extractor.NewMainNetExtractor(100, a.c.ExporterClient)
+
+	jetDrops := extractorMn.GetJetDrops(ctx)
+	refs := make([]types.Reference, 0)
+	for i := 0; i < recordsCount; i++ {
+		select {
+		case jd := <-jetDrops:
+			transform, err := transformer.Transform(ctx, jd)
+			if err != nil {
+				a.T().Logf("error transforming record: %v", err)
+				return
+			}
+			for _, t := range transform {
+				refs = append(refs, t.MainSection.Records[0].Ref)
+			}
+		}
+	}
+
+	transformerMn := transformer.NewMainNetTransformer(jetDrops)
+	err = transformerMn.Start(ctx)
+	require.NoError(a.T(), err)
+	defer transformerMn.Stop(ctx)
+
+	storage := storage.NewStorage(a.c.DB)
+	proc := processor.NewProcessor(transformerMn, storage, 1)
+	proc.Start(ctx)
+	defer proc.Stop(ctx)
+
+	time.Sleep(5 * time.Second)
```

Comment:
Can we wait without sleep here? Maybe wait for some event?

Reply:
I'll add waiters